src/pages/ReimbursementAccount/RequestorOnfidoStep.js

```diff
@@ -31,6 +31,16 @@ class RequestorOnfidoStep extends React.Component {
     constructor(props) {
         super(props);
         this.submit = this.submit.bind(this);
+        this.goBack = this.goBack.bind(this);
+    }
+
+    goBack(...args) {
```

Comment:
Didn't saw that one, `onBackButtonPress` is called inside `HeaderWithBackButton` without any args, so this spread is redundant in my opinion

Reply:
Sorry for this

Reply:
fixed